### ISSUE TITLE
Facing an issue in react-native storybookV - 7.6.17

### ISSUE THREAD

**Issue by kishore4a2** [eliasm307/storybook-addon-deep-controls#10]
Hi - i'm facing an issue after integrating with react-native with storybook version 7.6.17 and here is my code

````
const meta = {
  title: 'Label',
  component: Label,
  parameters: {
    deepControls: {
      layout: 'centered',
      enabled: true,
    },
  },
  argTypes: {
    'startIcon.url': { control: 'text' },
  },
} satisfies TypeWithDeepControls<Meta<typeof Label>>;

export default meta;

type Story = TypeWithDeepControls<StoryObj<typeof meta>>;

export const LabelViewControlPlayground: Story = {
  parameters: {
    deepControls: {
      layout: 'centered',
      enabled: true,
    },
  },
  args: {
    startIcon: {
      url: 'enter url here',
    },
  },
  argTypes: {
    'startIcon.url': { control: 'text', defaultValue: 'enter url here' },
  },
};
````
<img width="705" alt="image" src="https://github.com/eliasm307/storybook-addon-deep-controls/assets/4446311/c835e8e4-bf78-431c-ad05-deacb628a3af">

If you see the above one - startIcon control is displaying two times 
one is startIcon and the startIcon.url - if i enter any text in startIcon.url its taking startIcon - question here is why its displaying two controls ? and starticon.url is not picking the value that i enter.. Thanks in advance

**Comment by eliasm307** [comment]
Hi, thanks for raising this issue.

I'll look into it. I don't use react native so I'll need to do some investigation

**Comment by eliasm307** [comment]
@kishore4a2 would you be able to make a minimal github repo that reproduces this issue, so I am able to just clone it and investigate the issue? Might be easier than me trying to setup react native with storybook, which might be different from your setup

**Comment by eliasm307** [comment]
no response for a while, I assume this will be fixed via #24
